docs/topics/native/native-objc-interop.md

```diff
@@ -228,7 +228,11 @@ A Swift/Objective-C initializer is imported to Kotlin as constructors or as fact
 The latter happens with initializers declared in the Objective-C category or as a Swift extension,
 because Kotlin has no concept of extension constructors.
 
-Kotlin constructors are imported as initializers to Swift/Objective-C. 
+> Remember to expose Swift methods with `@objc` before importing Swift initializers to Kotlin.
```

Comment:
(Not sure why this tip is needed at all, but)
Let's replace "expose" with "annotate"?

Reply:
oh, sorry I forgot about that, there was another request ([KT-45753](https://youtrack.jetbrains.com/issue/KT-45753)), and `@sbogolepov` suggested just to add a note to clarify things